src/clients/github.js

```diff
@@ -0,0 +1,280 @@
+import get from 'lodash/get';
+import GitHub from 'github-api';
+import isEmpty from 'lodash/isEmpty';
+import trim from 'lodash/trim';
+import performWithRetries from '../util/performWithRetries';
+import generatePreview from '../util/generatePreview';
+
+const _anonymousGithub = new GitHub({});
+const COMMIT_MESSAGE = 'Created using Popcode: https://popcode.org';
+const MASTER = 'master';
+const GH_PAGES = 'gh-pages';
+const NETWORK_ERROR = 'Network Error';
+const _parser = new DOMParser();
+
+function EmptyGistError(message) {
+  this.name = 'EmptyGistError';
+  this.message = message;
+  this.stack = new Error().stack;
+}
+EmptyGistError.prototype = Object.create(Error.prototype);
+
+function _getPageTitle(preview) {
+  return _parser.parseFromString(
```

Comment:
Is there? I looked to see if #386 had been resolved yet, figuring I could use whatever they did, but it has not been.